### PR TITLE
Feat/better onboarding

### PR DESCRIPTION
**Pull request by GoOz**
Few improvements:

- [x] Added a `label` before the Cozy address' `input`.
- [x] More explicite example & errors.
- [x] The whole compliant with Cozy's styleguide.
- [x] ~~Added a "**https://**" prefix in front of the `input` and removed the auto-updated URL value to make the protocol implicit.~~
- [x] Select & focus the field when there's an error.
- [x] Added a spinner while we're fetching the server and its availability.

Basically it looks now like this:
onboarding

### CODE REVIEW

File: src/drive/mobile/containers/onboarding/SelectServer.jsx [hunk 1]
```diff
@@ -37,39 +64,46 @@ export class SelectServer extends Component {
...
           )}
         </div>
         <footer className={styles['wizard-footer']}>
           <button
             role="button"
             className={classNames(styles['c-btn'], styles['c-btn--regular'])}
-            onClick={() => selectServer(serverUrl)}
+            onClick={() => this.setServerUrl(serverUrl)}
             disabled={error || !serverUrl}
+            aria-busy={fetching ? true : false}
```
Comment (GoOz) [inline src/drive/mobile/containers/onboarding/SelectServer.jsx]: I first wrote `aria-busy={fetching}` but I wondered if it was explicit enough. You tell me.

File: src/drive/mobile/containers/onboarding/SelectServer.jsx [hunk 2]
```diff
@@ -9,10 +9,36 @@ import { registerDevice, setUrl } from '../../actions/settings'
 import styles from '../../styles/onboarding'
 
 export class SelectServer extends Component {
+  constructor(props) {
```
Comment (y-lohse) [inline src/drive/mobile/containers/onboarding/SelectServer.jsx]: I forgot about this. Actually, you can just do

```
export class SelectServer extends Component {
state = {fetching: false}
```

and then you don't even need a constructor.

File: src/drive/mobile/containers/onboarding/SelectServer.jsx [hunk 3]
```diff
@@ -9,10 +9,36 @@ import { registerDevice, setUrl } from '../../actions/settings'
...
+  }
+
   componentDidMount() {
     this.serverInput.focus()
   }
 
+  componentDidUpdate() {
+    if (this.props.error) {
+      this.serverInput.focus()
+      this.serverInput.select()
```
Comment (y-lohse) [inline src/drive/mobile/containers/onboarding/SelectServer.jsx]: What does `.select()` do?

File: src/drive/mobile/containers/onboarding/SelectServer.jsx (hunk 2 above)
Comment (GoOz) [reply src/drive/mobile/containers/onboarding/SelectServer.jsx]: oh right ok then :)

File: src/drive/mobile/containers/onboarding/SelectServer.jsx (hunk 3 above)
Comment (GoOz) [reply src/drive/mobile/containers/onboarding/SelectServer.jsx]: It selects the text in the input. Unless I was mislead, `.focus()` just put the cursor inside the field, ready to write and `select()` selects the string, ready to delete. 
I was asked to add this feature. :)

Comment (y-lohse) [reply src/drive/mobile/containers/onboarding/SelectServer.jsx]: Cool, makes sense then!

**Comment by GoOz** [comment]
[(1 + 0,5j) 📱 Améliorer l'écran d'erreur de la saisie de l'adresse du Cozy pour indiquer qu'il est possible de modifier l'adresse](https://trello.com/c/9Hun2kIg/563-1-05j-%F0%9F%93%B1-am%C3%A9liorer-l%C3%A9cran-derreur-de-la-saisie-de-ladresse-du-cozy-pour-indiquer-quil-est-possible-de-modifier-ladresse)
